app/views/admin/users/index.html.haml

```diff
@@ -23,17 +23,27 @@
           = user.last_login_at.strftime('%d.%m.%Y, %H:%M')
       %td= user.last_login_from
       %td= user.ldap_uid.to_s
+
       %td
         - if user != current_user
-          = change_admin_attribute(user)
+          .btn-group{id: user.id } 
+            %button.btn.btn-default.dropdown-toggle{id: 'role-dropdown','data-toggle':'dropdown'}
```

Comment:
move to helper as mentioned before

Reply:
http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag